File: 462-minimum-moves-to-equal-array-elements-ii/462-minimum-moves-to-equal-array-elements-ii.cpp
class Solution {
public:
    int minMoves2(vector<int>& nums) {
        
    sort(nums.begin() , nums.end()) ; 
    int ans = 0 , N = nums.size() ; 
    for( int i=0 ; i < N/2 ; i++){
        ans += (nums[N-1-i] -nums[i] ); 
    } 
    
        return ans; 


    }
};

File: 462-minimum-moves-to-equal-array-elements-ii/462-minimum-moves-to-equal-array-elements-ii.py
class Solution:
    def minMoves2(self, nums: List[int]) -> int:
        
        nums.sort() 
        sumv = 0 
        l , r = 0, len(nums)-1 
        while l<r: 
            sumv += nums[r]-nums[l] 
            l +=1 ; r -=1 

        return sumv 